index.js

```diff
@@ -478,23 +483,46 @@ function modifyFirstrun() {
     });
 }
 
+/**
+ * Overrides time interval defauls from config file.
+ */
+function overrideDefaults() {
+    try {
+        // load the config file
+        let config = self.data.load('./config.json');
+
+        // if the file existed, parse the contents to a JSON object
+        if (config) {
+            config = JSON.parse(config);
+            // override time intervals with values from config
+            defaultSidebarInterval = config.defaultSidebarInterval;
+            timeElapsedFormula = config.timeElapsedFormula;
+            waitInterval = config.waitInterval;
+        }
+    } catch(e) {
+        console.error(e);
```

Comment:
You will never see this in a production install but, no reason we cannot be explicit.